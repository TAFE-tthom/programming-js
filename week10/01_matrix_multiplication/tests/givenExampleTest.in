3 2 5 3
5 3 10
7 9 9
10 2 3 8 12
7 4 11 3 6
8 8 1 14 3
