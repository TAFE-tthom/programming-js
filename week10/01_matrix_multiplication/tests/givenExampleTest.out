151 102 58 189 108
205 122 129 209 165
